apps/heft/src/plugins/ApiExtractorPlugin/ApiExtractorPlugin.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { FileSystem, Terminal } from '@rushstack/node-core-library';
+
+import { IHeftPlugin } from '../../pluginFramework/IHeftPlugin';
+import { HeftSession } from '../../pluginFramework/HeftSession';
+import { HeftConfiguration } from '../../configuration/HeftConfiguration';
+import { IBundleStage, IBuildActionContext } from '../../cli/actions/BuildAction';
+import { ApiExtractorRunner } from './ApiExtractorRunner';
+
+const PLUGIN_NAME: string = 'ApiExtractorPlugin';
+const CONFIG_FILE_LOCATION: string = './.heft/api-extractor.json';
```

Comment:
For debugging investigations, it's very important than running `api-extractor run` manually will work without any special setup, and that it will repro any issues that Heft would encounter when invoking API Extractor programmatically.  The manual invocation provides easy access to switches like `--debug` and `--diagnostics`, and it is much easier to attach a debugger to.

Reply:
You'd just need to include `-c .heft/api-extractor.json` to the normal AE command line.

Reply:
Moved it back to config